influxql/parser_test.go

```diff
@@ -338,7 +337,7 @@ func TestParser_ParseStatement(t *testing.T) {
 
 		// CREATE USER statement
 		{
-			s: `CREATE USER testuser WITH PASSWORD pwd1337`,
+			s: `CREATE USER testuser WITH PASSWORD 'pwd1337'`,
```

Comment:
Users will need single quotes. It's a string. It's better than the SQL version which requires people to wrap the password in a call: `PASSWORD('xxx')`

http://dev.mysql.com/doc/refman/5.0/en/set-password.html